opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -15,78 +15,86 @@
 
 
 /**
- * Tracer is a simple, thin interface for Span creation, and Span propagation into different transport formats.
+ * Tracer is a simple, thin interface for Span creation and propagation across arbitrary transports.
  */
 public interface Tracer {
 
   /**
-   * Create, start, and return a new Span with the given `operationName`.
-   * An optional parent Span can be specified used to incorporate the newly-returned Span into an existing trace.
+   * Return a new SpanBuilder for a Span with the given `operationName`.
    *
-   * <p>Example:
+   * <p>A contrived example:
    * <pre>{@code
     Tracer tracer = ...
 
-    Span feed = tracer.buildSpan("GetFeed")
-                      .start();
+    Span parentSpan = tracer.buildSpan("DoWork")
+                            .start();
 
     Span http = tracer.buildSpan("HandleHTTPRequest")
-                      .withParent(feed)
+                      .asChildOf(parentSpan.context())
```

Comment:
@bensigelman similar to Node/Python, we can overload this to `asChildOf(parentSpan)`